Common/Tests/Utilities.UI/UI/ReplWindowProxy.cs

```diff
@@ -349,6 +411,22 @@ string executionMode
                 .Select(l => l.TrimEnd('\r', '\n', ' '))
                 .ToList();
 
+#if DEV14_OR_LATER
+            dynamic eval = _window.Evaluator;
+            var primary = eval.PrimaryPrompt as string ?? ">>>";
+#else
```

Comment:
@zooba, what's our general policy on this, actually? When changes have to be made to code for the sake of Python 2.5 support or older VS versions?